infrastructure/main.tf

```diff
@@ -12,167 +12,17 @@ data "azurerm_key_vault" "sscs_key_vault" {
-}
-
-data "azurerm_key_vault_secret" "idam-sscs-systemupdate-user" {
-  name = "idam-sscs-systemupdate-user"
-  vault_uri = "${data.azurerm_key_vault.sscs_key_vault.vault_uri}"
-}
-
-data "azurerm_key_vault_secret" "idam-sscs-systemupdate-password" {
-  name = "idam-sscs-systemupdate-password"
-  vault_uri = "${data.azurerm_key_vault.sscs_key_vault.vault_uri}"
-}
-
-data "azurerm_key_vault_secret" "idam-sscs-oauth2-client-secret" {
-  name = "idam-sscs-oauth2-client-secret"
-  vault_uri = "${data.azurerm_key_vault.sscs_key_vault.vault_uri}"
-}
-
-data "azurerm_key_vault_secret" "idam-api" {
-  name = "idam-api"
-  vault_uri = "${data.azurerm_key_vault.sscs_key_vault.vault_uri}"
-}
-
-data "azurerm_key_vault_secret" "sscs-notify-api-key" {
-  name = "notification-key"
-  vault_uri = "${data.azurerm_key_vault.sscs_key_vault.vault_uri}"
-}
-
-data "azurerm_key_vault_secret" "sscs-notify-api-test-key" {
-  name = "notification-test-key"
-  vault_uri = "${data.azurerm_key_vault.sscs_key_vault.vault_uri}"
-}
-
-data "azurerm_key_vault_secret" "email-mac-secret" {
-  name = "sscs-email-mac-secret-text"
-  vault_uri = "${data.azurerm_key_vault.sscs_key_vault.vault_uri}"
-}
-
-data "azurerm_key_vault_secret" "pdf_service_base_url" {
-  name      = "docmosis-endpoint"
-  vault_uri = "${data.azurerm_key_vault.sscs_key_vault.vault_uri}"
-}
-
-data "azurerm_key_vault_secret" "pdf_service_access_key" {
-  name      = "docmosis-api-key"
-  vault_uri = "${data.azurerm_key_vault.sscs_key_vault.vault_uri}"
-}
-
-data "azurerm_key_vault_secret" "sscs_asb_primary_send_and_listen_shared_access_key" {
-  name      = "sscs-asb-primary-send-and-listen-shared-access-key"
-  vault_uri = "${data.azurerm_key_vault.sscs_key_vault.vault_uri}"
-}
-
-data "azurerm_key_vault_secret" "appinsights_instrumentation_key" {
-  name      = "AppInsightsInstrumentationKey"
-  vault_uri = "${data.azurerm_key_vault.sscs_key_vault.vault_uri}"
-}
-
 resource "azurerm_key_vault_secret" "notification_job_scheduler_db_password" {
   name         = "notification-job-scheduler-db-password"
   value        = "${module.db-notif.postgresql_password}"
-  vault_uri = "${data.azurerm_key_vault.sscs_key_vault.vault_uri}"
+  key_vault_id = data.azurerm_key_vault.sscs_key_vault.id
 }
 
 locals {
-  local_ase = "${data.terraform_remote_state.core_apps_compute.ase_name[0]}"
-
-  ccdApi    = "http://ccd-data-store-api-${var.env}.service.${local.local_ase}.internal"
-  s2sCnpUrl = "http://rpe-service-auth-provider-${var.env}.service.${local.local_ase}.internal"
-  cohApi    = "http://coh-cor-${var.env}.service.${local.local_ase}.internal"
-  documentStore = "http://dm-store-${var.env}.service.${local.local_ase}.internal"
-  pdfService    = "http://cmc-pdf-service-${var.env}.service.${local.local_ase}.internal"
-
   azureVaultName = "sscs-${var.env}"
 }
 
-module "track-your-appeal-notifications" {
-  source       = "git@github.com:hmcts/cnp-module-webapp?ref=master"
```

Comment:
This is possible to remove straight away because we have RG creation explicitly. Should be used carefully on other repos. https://github.com/hmcts/sscs-track-your-appeal-notifications/pull/863/files#diff-1031eeab417cd8afe77e7e005717d38dL1